### DISCUSSION TITLE
Newbie query

### DISCUSSION THREAD

**Discussion by scorpi0n81** [TheLastBen/fast-stable-diffusion#2233] [Q&A] [answered]
I am using Colab Pro. Few newbie queries hence reaching out here

1. Is it required to execute every step from connect drive to start SD every single time? 
2. How to update the notebbok? If i copy the latest notebook to drive and replace with existing - does it impact models and lora stored in my GDrive under folder SD/webui/ models...?
3. Is there any way way to launch SD faster? Currently i run every step whenever i login to colab and it takes a lot of time in last step of starting SD.. (i am making it load 3 models from gdrive)

thanks

**Comment by TheLastBen** [comment]
Everytime you start a colab session, you need to run the cells from the top, but no need to redownload the models.
If you update the notebook, it won't affect your existing models and extensions.

Colab is relatively slow, that's why I recommend RunPod, you can get sd started in under 50 seconds

**Reply by scorpi0n81** [reply]
> Everytime you start a colab session, you need to run the cells from the top, but no need to redownload the models. If you update the notebook, it won't affect your existing models and extensions.
> 
> Colab is relatively slow, that's why I recommend RunPod, you can get sd started in under 50 seconds

Extremely helpful - thanks a ton, just a follow up - how do i check if the version of notebook i have is outdated or a newer version is available. Simply put - need something to understand if i need an update ?

**Reply by TheLastBen** [reply, marked as answer]
using the link https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb will always get you the latest notebook

update it regularly
